src/main/java/com/auth0/RequestProcessor.java

```diff
@@ -100,9 +105,9 @@ AuthorizeUrl buildAuthorizeUrl(HttpServletRequest request, String redirectUri, S
      *
      * @throws IdentityVerificationException if an error occurred while processing the request
      */
-    Tokens process(HttpServletRequest req) throws IdentityVerificationException {
+    Tokens process(HttpServletRequest req, HttpServletResponse response) throws IdentityVerificationException {
         assertNoError(req);
-        assertValidState(req);
+        assertValidState(req, response);
```

Comment:
nit, rename req to request ?

Reply:
Updated this and other usages of `req` to be `request`.